packages/modules/manage-versions/src/models/changeset.ts

```diff
@@ -6,7 +6,7 @@ export type Changeset = {
   id: string;
   displayName: string;
   description: string;
-  index: string;
+  index: number;
```

Comment:
because type is not supposed to be string.
https://developer.bentley.com/apis/imodels/operations/get-imodel-changesets/
![image](https://github.com/iTwin/admin-components-react/assets/148448690/1591b0a9-0843-430b-a3fc-59745142ac04)